tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/broadcast_audio.sh

```diff
@@ -43,6 +44,8 @@ done
 
 printf "\n\n======== Broadcaster sink disconnect test =========\n\n"
 
+SIMULATION_ID="broadcaster_sinkd"
```

Comment:
```suggestion
SIMULATION_ID="broadcaster_sink_disconnect"
```

Any reason not to use explicit naming?

Reply:
just shorter paths :), but I can change it right away

Reply:
I think explicit names in these tests are nice to have :)